### ISSUE TITLE
Version 1.5.13 breaks Github Actions

### ISSUE THREAD

**Issue by gvastakis** [GoogleCloudPlatform/cloud-spanner-emulator#152]
Hello, 

After updating to [version 1.5.13](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/releases/tag/v1.5.13) a Github Action responsible for my CI pipeline (that creates an emulator for integration tests) fails with the following error.

`ERROR: gcloud crashed (ConnectionError): HTTPConnectionPool(host='localhost', port=9020): Max retries exceeded with url: /v1/projects/emulator-project/instances?alt=json (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f036b291ed0>: Failed to establish a new connection: [Errno 111] Connection refused'))`

It seems the new version has some issue with specifying the ports ? Bellow is the related part of my ci.yml

```
    ..........
    services:
      emulator:
        image: gcr.io/cloud-spanner-emulator/emulator:1.5.12
        ports:
          - 9010:9010
          - 9020:9020
```

Before (v.1.5.12)

<img width="1383" alt="before" src="https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/assets/5776654/36744c44-39c0-471a-b6bf-ed97e327de59">

After (v.1.5.13)

<img width="1370" alt="after" src="https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/assets/5776654/3759979b-fdd3-4445-aa95-5b1e049f4d5d">

Reverting back to v.1.5.12 fixed the issue for me.

Thanks in advance,
Yannos

**Comment by evangelosdaniil** [comment]
We faced exactly the same issue! Downgrading the emulator to the previous version fixed it!

**Comment by panorama32** [comment]
I'm facing the same situation.
I couldn't connect to port 9010 and port 9020, even in the local environment, regardless of GitHub Actions.

**Comment by Barry-RG** [comment]
Having the same issues with it on CircleCI
Tried running the new image locally and it just instantly closes after printing 'Mocked Emulator'

**Comment by ssandler-panobi** [comment]
Same here, the update broke our CI tests. All tests time out when attempting to connect to the emulator admin client to create/list instances at the start of tests. Pinning at `1.5.12` fixed the problem for now.

**Comment by assadyousuf** [comment]
Had the same issue with circleCI. 1.5.12 works as stated above.

**Comment by yamadashy** [comment]
I've noticed a difference in the log outputs between versions 1.5.12 and 1.5.13 of the Google Cloud Spanner Emulator when running in Docker containers. Version 1.5.13 produces a more simplified log, while version 1.5.12 includes more detailed information.

Here are the specific differences in the logs:

In version 1.5.13, the log output is:
```shell
Mocked Emulator
```

In contrast, version 1.5.12 outputs detailed logs like:
```shell
2023/12/21 08:47:16 gateway.go:144: Cloud Spanner emulator running.
2023/12/21 08:47:16 gateway.go:145: REST server listening at 0.0.0.0:<port>
2023/12/21 08:47:16 gateway.go:146: gRPC server listening at 0.0.0.0:<port>
```

Upon further investigation, I found a relevant section in the [Google Cloud Spanner Emulator GitHub repository](https://github.com/GoogleCloudPlatform/cloud-spanner-emulator/blob/adf0ed09aa6f00327c5a04b319d0a31721f03d21/build/kokoro/gcp_ubuntu/release.sh#L50-L55) that might be related to this behavior. It seems that when the CLOUD_SPANNER_EMULATOR_MOCK_BUILD environment variable is set to true, the emulator outputs "Mocked Emulator."
